apps/accounts/tests/test_api.py

```diff
@@ -66,7 +66,8 @@ def test_user_self_get(self):
         self.assertEqual(response.status_code, 200)
         # Check if the content of the response corresponds to the expected json
         expected_json = {
-            'sub': user.username,
+            'sub': user.crosswalk.fhir_id,
+            'patient': user.crosswalk.fhir_id,
```

Comment:
Please confirm that you intended to change 'sub' to the fhir_id?

Reply:
yes, this is the "more" correct behavior for the userinfo endpoint.